website/content/docs/connect/ca/vault.mdx

```diff
@@ -181,124 +170,207 @@ The configuration options are listed below.
-	capabilities = [ "read" ]
-}
-```
+1. Add the following rules to your Vault policy file so that Consul
+   has read-only access to the root PKI engine, can automatically rotate
+   intermediate CAs as needed, and has full use of the intermediate PKI engine:
 
-</CodeBlockConfig>
+  <CodeBlockConfig filename="vault-managed-pki-policy.hcl">
 
-### Consul Managed PKI Paths
+  ```hcl
+  path "/<root_pki_path>/" {
+    capabilities = [ "read" ]
+  }
+  
+  path "/<root_pki_path>/root/sign-intermediate" {
+    capabilities = [ "update" ]
+  }
 
-The following Vault policy allows Consul to create and manage the PKI paths in Vault.
-Consul is granted the ability to create the PKI mount points and given full
-control of the Root and Intermediate or Leaf CA for Connect clients.
+  path "/<intermediate_pki_path>/*" {
+    capabilities = [ "create", "read", "update", "delete", "list" ]
+  }
+  ```
+  </CodeBlockConfig>
 
-In this example the `RootPKIPath` is `connect_root` and the `IntermediatePKIPath`
-is `connect_inter`. These values should be updated for your environment.
+1. Add the following rules to your Vault policy file so that Consul
+   can renew its Vault token if the token is renewable.
+   The rule enables the token to be renewed whether it is provided
+   [directly](#token) in the CA provider configuration or presented in an [auth method](#authmethod).
 
-<CodeBlockConfig filename="consul-managed-pki-policy.hcl">
+  <CodeBlockConfig filename="vault-managed-pki-policy.hcl">
 
-```hcl
-# Consul Managed PKI Mounts
-path "/sys/mounts" {
-  capabilities = [ "read" ]
-}
+  ```hcl
+  path "auth/token/renew-self" {
+    capabilities = [ "update" ]
+  }
 
-path "/sys/mounts/connect_root" {
-  capabilities = [ "create", "read", "update", "delete", "list" ]
-}
+  path "auth/token/lookup-self" {
+    capabilities = [ "read" ]
+  }
+  ```
+  
+  </CodeBlockConfig>
+
+1. Conditional and temporary: Only include the following rules in your Vault policy file
+   while you are changing the CA provider from Vault to a different CA provider (such as Consul's built-in provider)
+   or changing the `RootPKIPath` used by the Vault CA provider.
+   Those CA provider modifications trigger a root CA change that requires an
+   extremely privileged root cross-sign operation.
+   We recommend removing this elevated privilege immediately after the CA provider modification is finished.
+
+  <CodeBlockConfig filename="vault-managed-pki-policy.hcl">
+
+  ```hcl
+  path "<root_pki_path>/root/sign-self-issued" {
+    capabilities = [ "sudo", "update" ]
+  }
+  ```
+  </CodeBlockConfig>
 
-path "/sys/mounts/connect_inter" {
-  capabilities = [ "create", "read", "update", "delete", "list" ]
-}
+#### Define a policy for Consul-managed PKI paths ((#consul-managed-pki-paths))
 
-# Needed for Consul 1.11+
-path "/sys/mounts/connect_inter/tune" {
-  capabilities = [ "update" ]
-}
+To use Consul-managed PKI paths, ensure no PKI secrets engines are mounted at
```

Comment:
Same comment as the previous procedure - we already describe what each rule in the policy does in the steps, so I think we should remove the first two paragraphs in this introduction.